src/main/java/uk/gov/hmcts/ccd/config/JacksonObjectMapperConfig.java

```diff
@@ -19,10 +22,10 @@
     @Primary
     @Bean(name = "DefaultObjectMapper")
     public ObjectMapper defaultObjectMapper() {
-        ObjectMapper mapper = new ObjectMapper();
```

Comment:
this is the default mapper. Shall we ask some regression testing around this change?